src/runtime/nuxt-img.ts

```diff
@@ -83,15 +83,10 @@ export default {
       return
     }
 
-    // Get original image width & height
-    // TODO:
-    // const { width, height } = await this.$img.getResolution(this.src, {
-    //   provider: this.provider,
-    //   preset: this.preset
-    // })
-    const { width, height } = await this.$img.getMeta(this.src, {
+    const { width, height } = await this.$img.getResolution(this.src, {
       provider: this.provider,
-      preset: this.preset
+      preset: this.preset,
+      modifiers: this.modifiers
```

Comment:
Are the modifiers and preset required?

Reply:
Modifiers may change the size of the image, like `rotate`